docs/advancedmode.rst

```diff
@@ -0,0 +1,25 @@
+Advanced Mode
+=============
+
+CastleCMS by default hides several buttons/elements that most users may not need. These
+elements can be re-enabled by toggling "Advanced Mode."
+
+
+Toggling Advanced Mode
+----------------------
+
+Currently, to toggle Advanced Mode, there's a button on the Dexterity type list page. It
+is located there for now since everything that it shows/hides is related to Dextirity,
```

Comment:
- typo: "Dextirity"
- mention that advanced mode is stored in a cookie or that it is per-user, per-browser?